frontend/src/screens/AddCourse/services/course.service.ts

```diff
@@ -2,47 +2,49 @@ import { callApi } from 'helpers/api.helper';
 import { ICourse } from '../models/ICourse';
 import { ISaveLecture } from '../components/UploadLectureModal';
 
-export const getLecturesByAuthor = async (id: string) => {
+export const getLecturesByUser = async () => {
   const response = await callApi({
-    endpoint: `api/course/lectures/user/${id}`,
+    endpoint: `api/lecture/user`,
     type: 'GET'
   });
   const result = response.json();
   return result;
 };
 
-/* export const getLecturesByAuthor = async (id: string) => new Promise(resolve => {
-  setTimeout(() => resolve({
-    lectures
-  }), 500);
-}); */
-
 export const saveCourse = async (course: ICourse) => {
   const response = await callApi({
     type: 'POST',
-    endpoint: 'api/course',
+    endpoint: 'api/course/create',
     requestData: course
   });
   return response.json();
 };
 
+// export const saveLectureVideo = async lecture => {
+//   const request = { id: lecture.id, duration: lecture.duration };
+//   const response = await callApi({
+//     type: 'POST',
+//     endpoint: 'api/lecture/upload',
+//     requestData: request,
+//     attachment: lecture.video
+//   });
+//   return response.json();
+// };
```

Comment:
Do you need these comments?

Reply:
It's old method. With Multipart file atachment and request json body. I left it due to performance uncertainty of current request method. if it would work correctly, in next pr i'll remove old